docs/notebooks/Harmonic_Bias_PySAGES_OpenMM.md

```diff
@@ -0,0 +1,206 @@
+Now we can finally install PySAGES. We clone the newest version from [here](https://github.com/SSAGESLabs/PySAGES) and build the remaining pure python dependencies and PySAGES itself.
+<!-- #endregion -->
+
+```bash id="xYRGOcFJjEE6"
+
+rm -rf PySAGES
+git clone https://github.com/SSAGESLabs/PySAGES.git &> /dev/null
+cd PySAGES
+pip install -q . &> /dev/null
+```
+
+<!-- #region id="h5xD1zfj-J2z" -->
+#Harmonic Bias simulations
+<!-- #endregion -->
+
+```bash id="OIyRfOU9_cEJ"
+
+mkdir /content/harmonic-bias
+cd /content/harmonic-bias
+```
+
+<!-- #region id="Uh2y2RXDDZub" -->
+A harmonic bias simulation constraints a collective variable with a harmonic potential. This is useful for a variety of advanced sampling methods, in particular, umbrella sampling.
+ 
+For this colab, we are using Alanine-Dipeptide as the example molecule. So first, we fetch the molecule from the examples of PySAGES.
+<!-- #endregion -->
+
+```bash id="5fxJMNyE-RdA"
+
+cp /content/PySAGES/examples/abf/alanine-dipeptide-explicit.pdb ./
+```
+
+<!-- #region id="SqaG8YdK__FU" -->
+Next we load the PySAGES/OpenMM environment.
+<!-- #endregion -->
+
+```python colab={"base_uri": "https://localhost:8080/"} id="P6kPLtGI_-66" outputId="98e496cb-b78d-47bf-8b96-f2af942b10fc"
+from pysages.collective_variables import DihedralAngle
+from pysages.methods import HarmonicBias, HistogramLogger
+import numpy as np
+from pysages.utils import try_import
+
+import pysages
+
+openmm = try_import("openmm", "simtk.openmm")
+unit = try_import("openmm.unit", "simtk.unit")
+app = try_import("openmm.app", "simtk.openmm.app")
+```
+
+<!-- #region id="3TV4h_WEAdSm" -->
+Next, we write a function that can generate an execution context for OpenMM. This is everything you would normally write in an OpenMM script, just wrapped as a function that returns the context of the simulation.
+<!-- #endregion -->
+
+```python id="GAGw0s_cAcgP"
+def generate_simulation(**kwargs):
+    pdb_filename = "alanine-dipeptide-explicit.pdb"
+    T = 298.15 * unit.kelvin
+    dt = 2.0 * unit.femtoseconds
+    pdb = app.PDBFile(pdb_filename)
+
+    ff = app.ForceField("amber99sb.xml", "tip3p.xml")
+    cutoff_distance = 1.0 * unit.nanometer
+    topology = pdb.topology
+    system = ff.createSystem(
+        topology, constraints = app.HBonds, nonbondedMethod = app.NoCutoff,
+        nonbondedCutoff = cutoff_distance
+    )
+    
+    positions = pdb.getPositions(asNumpy = True)
+
+    integrator = openmm.LangevinIntegrator(T, 1 / unit.picosecond, dt)
+
+    simulation = app.Simulation(topology, system, integrator)
+    simulation.context.setPositions(positions)
+    simulation.minimizeEnergy()
+
+    return simulation
+```
+
+<!-- #region id="YtUoUMEdKtH8" -->
+The next step is to define the collective variable. In this case, we choose the two dihedral angles on the molecule as defined by its atom positions. We also choose the fixed center point for the harmonic bias simulation and the corresponding spring constant.
```

Comment:
"fixed center point" is a slightly weird formulation for me. What about "We also choose the equilibrium value at which to constrain the dihedrals...."